src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/metadata/ProjectSampleMetadataController.java

```diff
@@ -79,6 +79,21 @@ public String getProjectSamplesMetadataUploadPage(final Model model, @PathVariab
 		return "projects/project_samples_metadata_upload";
 	}
 
+	/**
```

Comment:
Can you move this into the `ProjectsController`. The reason for this is that as we move all the logic out into seperate AJAX controllers all that is left is simple and clean page request handlers.  Since these are small I want to keep them all in one file.

Reply:
Please see f2793323c3703ae37c8e3fcc5080336532bce7ea.